app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -6,9 +6,13 @@ CONFIG_INTEL_DMIC=n
 CONFIG_INTEL_DMIC_NHLT=n
 CONFIG_INTEL_SSP=n
 CONFIG_INTEL_ALH=n
+
+# enable additinal audio processing
 CONFIG_COMP_SRC=y
 CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
 CONFIG_COMP_DRC=y
+CONFIG_COMP_CROSSOVER=y
+CONFIG_COMP_MULTIBAND_DRC=y
```

Comment:
@lgirdwood @kv2019i Do you have opinion about this? If we are not having issues with .text size this might be acceptable. But if we should be more compact these could be default no. DRC is essential as are the EQs, these are more like optional for more advanced speaker enhance. What do you think?

Reply:
Lets have these in default Linux Kconfig.